SETUP.md

```diff
@@ -127,6 +128,14 @@ If you wish to do Verilog RTL Synthesis/create CircuitVerse Verilog Circuits in
       ```sh
       bin/yosys
       ```
```

Comment:
These steps are of a different service-  yosys2digitaljs-server, 

This must go under :

[CircuitVerse API documentation](https://github.com/CircuitVerse/CircuitVerse/blob/00e6ca84538071fd5308126a868dca562558a23e/SETUP.md#circuitverse-api-documentation) heading.